### ISSUE TITLE
Accessibility issues for complex tables

### ISSUE THREAD

**Issue by radu-pisoi** [dita-ot/dita-ot#2962]
## Expected Behavior
Table entries marked as table header using the ``scope`` attribute should be generated as table header (``th``). 
Also the value of the scope attribute should be transferred to HTML content.

## Steps to Reproduce
To reproduce the issue, you can use samples from DITA 1.3 specification:
http://docs.oasis-open.org/dita/dita/v1.3/errata01/os/complete/part3-all-inclusive/langRef/base/table.html#table

See table 2, with the _Sample with two header columns_ title.
```
<table frame="all">
 <title>Sample with two header columns</title>
 <tgroup cols="5">
  <colspec colname="c1"/>
  <colspec colname="c2"/>
  <colspec colname="c3"/>
  <colspec colname="c4"/>
  <colspec colname="c5"/>
  <thead>
   <row>
    <entry namest="c1" nameend="c2">Name</entry>
    <entry>Mark</entry>
    <entry>Peter</entry>
    <entry>Cindy</entry>
   </row>
  </thead>
  <tbody>
   <row>
    <entry morerows="1" scope="rowgroup"><b>points</b></entry>
    <entry scope="row"><b>expected</b></entry>
    <entry>10,000</entry>
    <entry>9,000</entry>
    <entry>10,000</entry>
   </row>
   <row>
    <entry scope="row"><b>actual</b></entry>
    <entry>11,123.45</entry>
    <entry>11,012.34</entry>
    <entry>10,987.64</entry>
   </row>
  </tbody>
 </tgroup>
</table>
```	

In the HTML5 output, I expect that cells ``points``, ``expected`` and ``actual`` to be generated using the ``th`` element. Also the _scope_ information should be transferred.

This issue is related with [Success Criterion 1.3.1 - Info and Relationships](https://www.w3.org/WAI/GL/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html) (Level A), see [H63: Using the scope attribute to associate header cells and data cells in data tables](https://www.w3.org/TR/WCAG20-TECHS/H63.html) technique.

* DITA-OT version: 3.0.2
* How did you run DITA-OT?
  oXygen
* Transformation type:
  HTML5
